rules/solid/tests/Rector/Foreach_/ChangeNestedForeachIfsToEarlyContinueRector/Fixture/multi_exprs_with_bool_oprnds.php.inc

```diff
@@ -45,7 +48,7 @@ class MultiExprsWithBoolOprnds implements RunnableInterface
             if (is_object($partPackage)) {
                 continue;
             }
-            if (1 == '1' && 2 == 't') {
+            if (!(1 == '1' || 2 == 't')) {
```

Comment:
Maybe keeping original code from here on would be better. What do you think?

Reply:
I'm not sure what you mean. If I undo the change, the rector output will not match and the test will not pass.

Reply:
Like this:

```diff
foreach( $partPackageList as $partPackage )
        {
-            if(true)
+            if(!true) {
+				continue;
			 }

-             {
                if (1 === 1 || 2 === 2)
                {
                    $payload[] = $partPackage;
                }
-            }
        }
```